lib/utils.js

```diff
@@ -21,7 +21,7 @@ function getInterfaceLanguage() {
 function validateTranslationKeys(translationKeys) {
   var reservedNames = ['_interfaceLanguage', '_language', '_defaultLanguage', '_defaultLanguageFirstLevelKeys', '_props'];
   translationKeys.forEach(function (key) {
-    if (reservedNames.includes(key)) {
```

Comment:
Because this is not sources, but transpiled code.
This pitfall doesn't affect this code, what you can actually see from this file.